CHANGELOG.md

```diff
@@ -5,20 +5,27 @@
 ### Breaking Changes
 
 * qspi: updated from C to C++, and fixed up the API to be cleaner, and simpler to use
+* sdram: Create SdramHandle class. Gets rid of configurable pin and hardcodes it instead.
 
 ### Features
 
 * adc: added initialization mappings for pins ADC1_INP12 (PC2) and ADC1_INP13 (PC3) (Not accessible on Daisy Seed)
+* bootloader: BOOT_APP macro added for running a program from the daisy bootloader
+* bootloader: added a number of deinitialization methods to core classes
+* bootloader: added `System` and `DaisySeed` code to check code location at runtime and initialize accordingly
+* bootloader: added `program-app` make target to upload code to the daisy bootloader
```

Comment:
I'd classify "features" as things that are user-accessible. so you could probably move these to the "other" section instead.

might be worth lumping most of these together into something like:

* bootloader: added deinit, and other groundwork for compatibility with Daisy Bootloader

We'll be making a wiki page dedicated to all things bootloader as we wrap up the project. So I don't think we need to bloat the changelog too much.